src/moments.jl

```diff
@@ -15,10 +15,12 @@ whereas it's `length(x)-1` in `Base.varm`. The impact is that this is not a
 weighted estimate of the population variance based on the sample; it's the weighted
 variance of the sample.
 """
-Base.varm(v::RealArray, wv::WeightVec, m::Real) = _moment2(v, wv, m)
+function Base.varm(v::RealArray, wv::AbstractWeights, m::Real; corrected=true)
```

Comment:
I don't really care. There's a short style guide in Julia's CONTRIBUTING.md, but I don't think it's mentioned.